Configuration.mk

```
@@ -73,12 +77,29 @@ TOCK_ARCHS := $(sort $(foreach target, $(TOCK_TARGETS), $(firstword $(subst |, ,
 
 # Check if elf2tab exists, if not, install it using cargo.
 ELF2TAB ?= elf2tab
+ELF2TAB_REQUIRED_VERSION := 0.8.0
```

```suggestion
ELF2TAB_REQUIRED_VERSION := 0.7.0
```
based on the similar change in the elf2tab repo

---

Sorry for that, I was under the impression that 0.70 is the current version.